### PR TITLE
Added the `root` command.

### PR DESCRIPTION
**Pull request by bpeabody**
Prints the root of the workdir of a meta-repo.

Addresses: https://github.com/twosigma/git-meta/issues/270

### CODE REVIEW

File: node/lib/cmd/root.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+/**
+ * Execute the `add` command according to the specified `args`.
+ *
+ * @async
+ * @param {Object}   args
+ * @param {String[]} args.paths
+ */
+exports.executeableSubcommand = function () {
```
Comment (jhedwardyang) [inline node/lib/cmd/root.js]: For similar style with other commands, can we have this return a generator wrapped in a Promise by co.wrap?

Comment (bpeabody) [inline node/lib/cmd/root.js]: I'd have to do something a little hacky to stop JSHint from complaining about a lack of `yield` in the body of the generator.

Comment (jhedwardyang) [inline node/lib/cmd/root.js]: I'm okay with this merged in the current state then.

**Comment by jhedwardyang** [comment]
Can you update this review to get Travis running on it? Thanks!

**Pull request merged**